samples/qml/requirements.txt

```diff
@@ -0,0 +1,2 @@
+PySide2==5.15.1
```

Comment:
Why do we need these?

Reply:
I created this just so that it is easy to install and build quickly.

Reply:
Ok, please add a comment somewhere explaining why it's easier and why it's needed. This is helpful for the final design.